Yoast/wordpress-seo
# P1-1282 - improve wincher integration on multisite

## Context


* The Wincher integration is not available on multisite. The integrations (tab) on Yoast SEO didn’t show the Wincher toggle and that made it very confusing.

## Summary


This PR can be summarized in the following changelog entry:

* Adds a disabled Wincher integration toggle to the Network admin/Multisite `integrations` tab.

## Relevant technical choices:

* Added a `upgrade routine` to set the Wincher toggle to 'Disable/Off' on Multisites/Network admin
  * Else it was possible that the toggle was disabled and still showed 'Allow Control/On'

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Testing using a multisite:
  * Click on `SEO -> General`  in the WordPress sidebar
  * Navigate to the `Integrations` tab
  * Verify the `Wincher integration` toggle is disabled and has the value 'Off'
  * Verify the copy reads: `Currently, the Wincher integration is not available for multisites.`
* Testing using the Network Admin:
  * Click on `SEO -> General`  in the WordPress sidebar
  * Navigate to the `Integrations` tab
  * Verify the `Wincher integration` toggle is disabled and has the value 'Disabled'
  * Verify the copy reads: `Currently, the Wincher integration is not available for multisites.`
* Testing using a `non multisite site`:
  * Click on `SEO -> General`  in the WordPress sidebar
  * Navigate to the `Integrations` tab
  * Verify the `Wincher integration` toggle is 'toggleable'
* Testing the `upgrade routine`
  * Access the multisite database and navigate to the '`*_sitemeta` table > `wpseo_ms` row'
  * Find and replace:
    * `s:32:"allow_wincher_integration_active";b:0;`  
    `s:32:"allow_wincher_integration_active";b:1;`
  * Go to the `Network Admin` dashboard
  * Click on `SEO -> General`  in the WordPress sidebar
  * Navigate to the `Integrations` tab
  * Verify the `Wincher integration` toggle is disabled and has the value `Allow Control`
  * Go to the dashboard of a multisite
  * Get the yoast-test-helper plugin
  * Go to tools>yoast-test-helper
  * Set the DB version to 18.6.
  * See that an upgrade routine is called.
  * Go to the `Network Admin` dashboard
  * Click on `SEO -> General`  in the WordPress sidebar
  * Navigate to the `Integrations` tab
  * Verify the `Wincher integration` toggle is disabled and has the value `Disable`
  * Verify in the database the `allow_wincher_integration_active` is set to **0**
* Testing the `upgrade routine` (**for when the code is in the RC**)
  * Have 18.5 
  * Access the multisite database and navigate to the '`*_sitemeta` table > `wpseo_ms` row'
  * Find and replace:
    * `s:32:"allow_wincher_integration_active";b:0;`  
    `s:32:"allow_wincher_integration_active";b:1;`
  * Go to the `Network Admin` dashboard
  * Click on `SEO -> General`  in the WordPress sidebar
  * Navigate to the `Integrations` tab
  * Verify the `Wincher integration` toggle is disabled and has the value `Allow Control`
  * Upgrade to 18.6
  * Go to the `Network Admin` dashboard
  * Click on `SEO -> General`  in the WordPress sidebar
  * Navigate to the `Integrations` tab
  * Verify the `Wincher integration` toggle is disabled and has the value `Disable`
  * Verify in the database the `allow_wincher_integration_active` is set to **0**

### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* 

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Other environments


## Documentation


## Quality assurance


Fixes P1-1282
